prog/prio.go

```diff
@@ -103,6 +103,8 @@ func (target *Target) calcResourceUsage() map[string]map[int]weights {
 				}
 			case BufferFilename:
 				noteUsage(uses, c, 10, DirIn, "filename")
+			case BufferGlob:
+				noteUsage(uses, c, 10, ctx.Dir, "glob")
```

Comment:
What's the reasoning behind this?

Reply:
Not show what it is, I just search BufferFilename and make the similar changes.

Reply:
This is not the best way to develop code :)